security-command-center/snippets/src/main/java/vtwo/source/GetSource.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo.source;
+
+// [START securitycenter_get_source_v2]
+
+import com.google.cloud.securitycenter.v2.GetSourceRequest;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+import com.google.cloud.securitycenter.v2.Source;
+import com.google.cloud.securitycenter.v2.SourceName;
+import java.io.IOException;
+
+public class GetSource {
+
+  public static void main(String[] args) {
+    // TODO: Replace the below variables.
+    // projectId: Google Cloud Project id.
+    String organizationId = "{google-cloud-organization-id}";
+
+    // Specify the source-id.
+    String sourceId = "{source-id}";
+
+    getSource(organizationId, sourceId);
+  }
+
+  // Demonstrates how to retrieve a specific source.
+  public static Source getSource(String organizationId, String sourceId) {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+
+      // Start setting up a request to get a source.
+      SourceName sourceName = SourceName.ofOrganizationSourceName(organizationId, sourceId);
```

Comment:
Doesn't the v2 API require a location to be passed as well?

Reply:
When I create a source, this is the response:
Created source : name: "organizations/**/sources/**"

To use V2, this syntax must be used:
"organizations/[organization_id]/source/[source_id]"

but I tried other approaches like
"organizations/%s/locations/%s/sources/%s"
"organizations/%s/sources/%s/locations/%s"
 SourceName sourceName = SourceName.ofOrganizationSourceName(organizationId, sourceId);
 Parent String = String.format("%s/locals/%s", sourceName.toString(), "global");

all resulting in an INVALID_ARGUMENT.
Kindly show me the best approach.